### PR TITLE
feat(language): added latin variation of serbian language (ekmas)

### PR DESCRIPTION
**Pull request by ekmas**
### Description

I added latin variation of serbian language, and I renamed the old serbian version to `serbian_cyrillic`.

### Checks

- [x] Adding a language or a theme?
    - [x] If is a language, did you edit `_list.json`, `_groups.json` and add `languages.json`?
- [x] Check if any open issues are related to this PR; if so, be sure to tag them below.
- [x] Make sure the PR title follows the Conventional Commits standard. (https://www.conventionalcommits.org for more info)
- [x] Make sure to include your GitHub username inside round brackets at the end of the PR title

### CODE REVIEW

File: frontend/static/languages/serbian_cyrillic.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "serbian",
+  "name": "serbian_cyrillic",
   "noLazyMode": true,
   "words": [
     "као",
```
Comment (Miodec) [inline frontend/static/languages/serbian_cyrillic.json:5]: Could you revert this change? It might cause some issues for people that use this language. Keep it `serbian` and `serbian_latin`.

**Review by Miodec — CHANGES_REQUESTED** [review]

File: frontend/static/languages/serbian_cyrillic.json (hunk 1 above)
Comment (ekmas) [reply frontend/static/languages/serbian_cyrillic.json:5]: should I only change `name` property or should I change `serbian_cyrillic` to `serbian` wherever `serbian_cyrillic` appears?
